ui/src/components/sermons/OnlinePageTabs.js

```diff
@@ -26,129 +32,222 @@ const OnlinePageTabs = ({ sermonNotes }) => {
     setNoteId(noteId + 1);
   };
 
-  return (
-    <Tabs
-      isFitted
-      h="100%"
-      onChange={(i) => setTab(i)} overflowY={(tab ? 'auto' : 'hidden')}
-    >
-      <TabList orientation="horizontal">
-        <Tab>Sermon Notes</Tab>
-        <Tab>New to HMCC</Tab>
+  const IgnitePage = () => {
```

Comment:
![image](https://user-images.githubusercontent.com/24607458/169099948-abb28a24-8a2b-461a-b820-9aa2a0d174ed.png)

may need to get rid of the scroll bar

Reply:
not a big deal since this isnt going into our main branch - but this component is not named properly. it's not a page we are rendered, but a tab (`IgniteTab` instead of `IgnitePage`)


also, this whole component has a lot of repeated code from the usual tab. I think it would've been much cleaner to insert the ignite components into the actual tab components instead of making a whole new functional component

Reply:
not a big deal since this isnt going into our main branch - but this component is not named properly. it's not a page we are rendered, but a tab (`IgniteTab` instead of `IgnitePage`)


also, this whole component has a lot of repeated code from the usual tab. I think it would've been much cleaner to insert the ignite components into the actual tab components instead of making a whole new functional component